### PR TITLE
Remove indirect reference to the util packages

### PR DESCRIPTION
**Pull request by kerthcet**
#### What type of PR is this?
/kind cleanup
/sig scheduling
/assign @ahg-g 

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
None
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Review by sanposhiho — COMMENTED** [review]
/lgtm
/retest

I completely agree with this change, but do we have any idea why we were doing this? (just curious)

File: test/integration/scheduler/plugins/plugins_test.go [hunk 1]
```diff
@@ -55,21 +55,24 @@ import (
 
 // imported from testutils
```
Comment (sanposhiho) [inline test/integration/scheduler/plugins/plugins_test.go:56]: nit: remove this comment

**Comment by kerthcet** [comment]
> I completely agree with this change, but do we have any idea why we were doing this? (just curious)

Yes, see discussion here https://github.com/kubernetes/kubernetes/pull/119046#discussion_r1252926505

File: test/integration/scheduler/plugins/plugins_test.go (hunk 1 above)
Comment (kerthcet) [reply test/integration/scheduler/plugins/plugins_test.go:56]: I'll follow this with another cleanup, thanks.

**Comment by ahg-g** [comment]
Thanks @kerthcet for following up

**Pull request merged**